### PR TITLE
BAEL-4302 : How can I list all classes loaded in a specific class loader

### PR DESCRIPTION
**Pull request by anmoldeep0123**

### CODE REVIEW

File: core-java-modules/core-java-jvm-2/src/main/java/com/baeldung/loadedclasslisting/ClassLoaderType.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package com.baeldung.loadedclasslisting;
+
+public enum ClassLoaderType {
+
```
Comment (KevinGilmore) [inline core-java-modules/core-java-jvm-2/src/main/java/com/baeldung/loadedclasslisting/ClassLoaderType.java:4]: remove blank line

File: core-java-modules/core-java-jvm-2/src/main/java/com/baeldung/loadedclasslisting/Launcher.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+    public static void main(String[] args) {
+        
+        printClassesLoadedBy(ClassLoaderType.BOOTSTRAP);
+
+        printClassesLoadedBy(ClassLoaderType.SYSTEM);
+
+        printClassesLoadedBy(ClassLoaderType.EXTENSION);
+
+        printClassesLoadedBy(ClassLoaderType.CUSTOM);
```
Comment (KevinGilmore) [inline core-java-modules/core-java-jvm-2/src/main/java/com/baeldung/loadedclasslisting/Launcher.java:21]: remove blank lines

File: core-java-modules/core-java-jvm-2/src/main/java/com/baeldung/loadedclasslisting/customLoader/ClassLoaderInfo.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package com.baeldung.loadedclasslisting.customLoader;
+
+import java.util.ArrayList;
+
+public class ClassLoaderInfo {
+
+    public void printClassLoaders() throws ClassNotFoundException {
+
```
Comment (KevinGilmore) [inline core-java-modules/core-java-jvm-2/src/main/java/com/baeldung/loadedclasslisting/customLoader/ClassLoaderInfo.java:8]: remove blank line

Comment (anmoldeep0123) [reply core-java-modules/core-java-jvm-2/src/main/java/com/baeldung/loadedclasslisting/customLoader/ClassLoaderInfo.java]: resolved

File: core-java-modules/core-java-jvm-2/src/main/java/com/baeldung/loadedclasslisting/Launcher.java (hunk 2 above)
Comment (anmoldeep0123) [reply core-java-modules/core-java-jvm-2/src/main/java/com/baeldung/loadedclasslisting/Launcher.java]: removed

File: core-java-modules/core-java-jvm-2/src/main/java/com/baeldung/loadedclasslisting/ClassLoaderType.java (hunk 1 above)
Comment (anmoldeep0123) [reply core-java-modules/core-java-jvm-2/src/main/java/com/baeldung/loadedclasslisting/ClassLoaderType.java]: removed this class ; refactored this to a public enum inside ListLoadedClassesAgent.java

**Comment by anmoldeep0123** [comment]
@KevinGilmore  , resolved the issues . Please check. Thanks

**Comment by anmoldeep0123** [comment]
@KevinGilmore  final changes, please review

**Comment by anmoldeep0123** [comment]
@KevinGilmore , any updates ??

**Pull request merged**